Library/Formula/uim.rb

```diff
@@ -1,17 +1,17 @@
-require 'formula'
-
 class Uim < Formula
   desc "Multilingual input method library"
-  homepage 'http://code.google.com/p/uim/'
-  url 'https://uim.googlecode.com/files/uim-1.6.0.tar.bz2'
-  sha1 'd27f2ca8136da0702c82f0522911d06b2b8f8ea7'
+  homepage "http://code.google.com/p/uim/"
+  url "https://github.com/uim/uim/releases/download/uim-1.8.6/uim-1.8.6.tar.gz"
+  sha256 "0eebcf5b2192d0cd11018adbd69e358bfa062d3b5d63c1dd0b73d5f63eb7afe7"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'gettext'
+  depends_on :x11
```

Comment:
I didn't become aware of it until I started working on it. I think it will indeed have to be moved over.